### PR TITLE
Pentagonal number

### PR DESCRIPTION
**Pull request by aberu78**

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -122,11 +128,21 @@ public static void main(String[] args) {
     }
 
     /***
-     * This fuction will take a string parameter and return 2d array of binary matrix.
+     * This functio will read a positive integer and determined how many dots exist in a pentagonal shape
```
Comment (roninsteven) [inline src/Main.java:131]: misspelled function check your typos :)

File: src/Main.java [hunk 2]
```diff
@@ -122,11 +128,21 @@ public static void main(String[] args) {
...
+
+        return (int)((5*Math.pow(n,2)) - 5*n +2) / 2;
+    }
+    /***
+     * This function will take a string parameter and return 2d array of binary matrix.
      * @param str The string that will change
      * @return newArr The 2d array of the str
      */
-    private static int[][]  matrix (String str){
+    private static int[][] matrix (String str){
```
Comment (roninsteven) [inline src/Main.java:145]: reformat file to add appropriate spaces ){

File: src/Main.java [hunk 3]
```diff
@@ -107,9 +107,15 @@ public static void main(String[] args) {
                     System.out.println("Enter String (Maximal Square challenge)");
                     str = scanner.nextLine();
 
-                    int matrix[][] = matrix(str);
+                    int[][] matrix = matrix(str);
```
Comment (roninsteven) [inline src/Main.java:110]: You asked here is the difference

They are semantically identical. The int array[] syntax was only added to help C programmers get used to java.
int[] array is much preferable and less confusing.

File: src/Main.java (hunk 1 above)
Comment (roninsteven) [inline src/Main.java:131]: misspelled function check your typos :)

File: src/Main.java [hunk 4]
```diff
@@ -333,24 +349,23 @@ public static int simpleAdding(int num) {
...
         String changedVar = "";
-        char aWord[] = str.toCharArray();
+        char[] aWord = str.toCharArray();
         for (int i = 0; i < aWord.length; i++) {
             if (Character.isLetter(aWord[i])) {
-                int ascii = (int) aWord[i];
+                int ascii = aWord[i];
 
                 if (ascii != (int) 'z' && ascii != 'Z') {
                     ascii += 1;
```
Comment (roninsteven) [inline src/Main.java:371]: ascii++

**Review by roninsteven — APPROVED** [review]
Looks good :)

File: src/Main.java (hunk 2 above)
Comment (aberu78) [reply src/Main.java:145]: done. thank you

File: src/Main.java (hunk 3 above)
Comment (aberu78) [reply src/Main.java:110]: thank you for info

File: src/Main.java (hunk 1 above)
Comment (aberu78) [reply src/Main.java:131]: fixed it. thank you

File: src/Main.java (hunk 4 above)
Comment (aberu78) [reply src/Main.java:371]: fixed it. thank you

**Pull request merged**